chainer_chemistry/models/prediction/base.py

```diff
@@ -134,6 +134,38 @@ def _forward(self, data, fn, batchsize=16,
     def save_pickle(self, filepath, protocol=None):
         """Save the model to `filepath` as a pickle file
 
+        This function send the parameters to CPU before saving the model so
+        that the pickled file can be loaded with in CPU-only environment. 
```

Comment:
I think it may remind users that this method has a side effect and parameters are in CPU after the method is called. But actually, it moves back to an original device. So, I would suggest to describe it.